ingest/validate_de.py

```diff
@@ -0,0 +1,141 @@
+import pandas as pd
+import itertools
+import numpy as np
+file_path = "/Users/mvelyuns/scp-ingest-pipeline/ingest/pairwise.csv"
+data = pd.read_csv(file_path)
```

Comment:
Does this handle TSVs, too?  ([We'll need to](https://docs.google.com/document/d/1R1wBgps4Vx0B98BERGfKIO9H6Qy3MNtSqKTwvAykL3A/edit#).)